### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by KingsleyIbe**
### In this milestone, I implemented the following:
- Added RSpec as a gem to the project.
- Created a Request spec file for all your controllers.
- Ensured that for each action the following were checked:
      - If response status was correct.
      - If a correct template was rendered.
      - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/controllers/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Users controller', type: :request do
```
Comment (hiromataba) [inline spec/controllers/users_spec.rb:3]: - [ ] Please put your post and user controllers specs in `spec/requests/` directory not `spec/controllers` directory,
Request specs allow you to focus on a single controller action, but unlike controller tests involve the router, the middleware stack, and both rack requests and responses. This adds realism to the test that you are writing, and helps avoid many of the issues that are common in controller specs

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @KingsleyIbe   :wave: :wave: ,

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_military: 
- No linter errors :heavy_check_mark: 
- Nice PR ttile and description :heavy_check_mark: 
- Nice commit history :heavy_check_mark: 

## Required Changes ♻️

- [ ] I am getting some errors running some tests please fix that

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/controllers/users_spec.rb (hunk 1 above)
Comment (KingsleyIbe) [reply spec/controllers/users_spec.rb:3]: Hey @hiromataba 

Thank you for your suggestions
Changes will be effected.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Kingsley, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review_
<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hey @rloterh 

Thank you for approving my project.

File: spec/controllers/users_spec.rb (hunk 1 above)
Comment (hiromataba) [reply spec/controllers/users_spec.rb:3]: You are welcome @KingsleyIbe :tada:

**Pull request merged**